src/pages/trade/HorizontalPanel.tsx

```diff
@@ -308,13 +309,16 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen }: ElementProps) => {
 const $AssetIcon = styled(AssetIcon)`
   font-size: 1.5em;
 `;
+const collapsibleTabsType = getSimpleStyledOutputType(CollapsibleTabs, {} as {});
+
 const $CollapsibleTabs = styled(CollapsibleTabs)`
   --tableHeader-backgroundColor: var(--color-layer-3);
 
   header {
     background-color: var(--color-layer-2);
   }
-` as typeof CollapsibleTabs<InfoSection>;
+` as typeof collapsibleTabsType;
```

Comment:
Do we completely lose the `<InfoSection>` generic?

Reply:
Nope, it gets to be inferred at usage-time, as all generics would prefer.